### PR TITLE
Update rubocop version

### PR DESCRIPTION
**Pull request by hmtanbir**
bumped rubocop from 0.60 to 0.76

### CODE REVIEW

**Comment by hmtanbir** [comment]
@damianlegawiec Would you please check my PR ?

File: common_spree_dependencies.rb [hunk 1]
```diff
@@ -36,7 +36,7 @@
 end
 
 group :test, :development do
-  gem 'rubocop', '~> 0.60.0', require: false
+  gem 'rubocop', '~> 0.77.0', require: false # bumped
```
Comment (damianlegawiec) [inline common_spree_dependencies.rb]:
```suggestion
  gem 'rubocop', '~> 0.76.0', require: false # bumped
```

We need to keep it at 0.76 as this is the last version supported by CodeClimatee - https://docs.codeclimate.com/docs/rubocop#section-using-rubocop-s-newer-versions

Also, we need to change the code climate to actually use this version here: https://github.com/spree/spree/blob/master/.codeclimate.yml#L7

Comment (hmtanbir) [reply common_spree_dependencies.rb]: @damianlegawiec I have updated the version, would you please check it ?

**Comment by damianlegawiec** [comment]
Thank you @hmtanbir - there are some new issues found by CodeClimate after this update - https://codeclimate.com/github/spree/spree/pull/9666 - could we exclude those via rubocop rules? We would like to keep that style for now

**Comment by hmtanbir** [comment]
@damianlegawiec Would you please review ?

**Comment by hmtanbir** [comment]
@mbajur @devilcoders Would you please check this PR ?

**Review by damianlegawiec — APPROVED** [review]

**Pull request merged**